.rubocop.yml

```diff
@@ -373,10 +379,37 @@ RSpec/ScatteredLet:
 RSpec/NestedGroups:
   Enabled: false
 RSpec/VariableName:
-  Exclude:
-    - 'spec/system/schools/advice_pages/hot_water_spec.rb'
+  AllowedPatterns:
+    - _£_
 RSpec/ScatteredSetup:
   Enabled: false
+RSpec/ItBehavesLike:
+  EnforcedStyle: 'it_behaves_like' # rather than 'it_should_behave_like'
+RSpec/MultipleExpectations:
+  Max: 12 # max expectations per test
+  Exclude:
```

Comment:
Are these manually added, rather than generated via `auto-gen-config`?

Reply:
Yes - because of wanting a config of Max: 12 (for now), and .rubocop.yml inheriting from  .rubocop_todo.yml, it seemed the only way to do it as far as I can tell.